.github/workflows/build_and_scan_images.yaml

```
@@ -0,0 +1,28 @@
+name: Build and scan OCI images
+
+on:
+  schedule:
+  # every day at 1:00AM UTC
+  - cron: '0 1 * * *'
```

Someone pointed me to [this blue box tip](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) from Github recently - they recommend you avoid scheduling jobs on the hour because they're high load times and can be delayed.  Not that it matters, but if you want to ensure it runs right away you can pick something slightly odder like this
```suggestion
  # every day at 1:12AM UTC
  - cron: '12 1 * * *'
```